src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        System.out.println(ball1.getRandomBall());
+        Lottery ball2 = new Lottery();
+        System.out.println(ball2.getRandomBall());
+        Lottery ball3 = new Lottery();
```

Comment:
numbers in naming = bad practice, most of the time it means you can automate the creation, like here: just create a for loop and call inside getRandomBall(), you don't even need to name the Lottery variable